source/general/time.rst

```diff
@@ -5,46 +5,88 @@
+
+* :ref:`iact-events`
+* :ref:`iact-gti`
+* :ref:`iact-pnt`
+* :ref:`obs-index`
+
+Other useful resources concerning time:
+
+* https://heasarc.gsfc.nasa.gov/docs/fcg/common_dict.html
+* `Time in Fermi data analysis`_
+
+.. _time-formats:
+
+Formats
+-------
+
+Times should be given as 64-bit float columns, relative to a reference time point
+that is specified by the following FITS header keywords:
 
-Times are used in many places in high-level analysis, e.g.
+* ``MJDREFI`` type: int, unit: days
+    * Integer part of instrument specific MJD time reference
+* ``MJDREFF`` type: float, unit: days
+    * Float part of instrument specific MJD time reference
+* ``TIMEUNIT`` type: string
+    * Time unit (e.g. 's')
+* ``TIMESYS`` type: string
+    * Time system, also referred as time scale (e.g. 'UT', 'UTC', 'TT', 'TAI')
+* ``TIMEREF`` type: string
+    * Time reference frame, used for example for barycentric corrections
+      (options: 'LOCAL', 'SOLARSYSTEM', 'HELIOCENTRIC', 'GEOCENTRIC')
 
-* Observations have start and end times and sometimes are split up into
-  "good time intervals" GTIs when hardware issues occur or clouds pass the field of view.
-* Gamma-ray events are observed at given times, and those times are needed
-  to convert the reconstructed AltAz position to RaDec, or to select events in
-  a given GTI.
-* Some gamma-ray sources are variable, e.g. AGNs can flare on timescales of seconds or minutes,
-  or pulsars emit a periodic signal on timescales of seconds or milli-seconds.
+See the `FITS standard`_ and the `FITS time paper`_ for further information.
 
-This page contains specifications and recommendations how to work with times
-for high-level gamma-ray astronomy, i.e. how to store times in files
-(e.g. event lists, GTI extension, observation tables) and
-take times as input and output in analysis tools.
+In addition to that main way of specifying times as a floating point number wrt. a reference timepoint,
+the following header keys with date and time values as strings can be added.
+This is for convenience and humans reading the information. Usually science tools will not access
+this redundant and optional information. The time system used should be the one given by ``TIMESYS``.
 
-Reference documents and tools
------------------------------
+* ``DATE-OBS`` type: string
+    * Observation start date (format: "yyyy-mm-dd")
+* ``TIME-OBS`` type: string
+    * Observation start time (format: "hh:mm:ss.sss...")
+* ``DATE-END`` type: string
+    * Observation end date (format: "yyyy-mm-dd")
+* ``TIME-END`` type: string
+    * Observation end time (format: "hh:mm:ss.sss...")
 
-Basically we follow `Time in Fermi data analysis`_, so this is the number one
-reference.
+Note that the FITS standard allows and it is quite common to instead put a
+``TIME-OBS`` key with value "yyyy-mm-ddThh:mm:ss.sss..." and to omit the ``DATE-OBS`` key
+(see `Dictionary of Commonly Used FITS Keywords`_). That is allowed as well.
```

Comment:
I would rephrase this paragraph a bit (is not too formal at the moment). At least I would remove the "That is allowed as well".